data/modules/Assassination/Assassination.lua

```diff
@@ -216,7 +217,7 @@ local makeAdvert = function (station)
 		reward = reward,
 		threat = threat,
 		shipid = hullConfig.id,
-		shipname = ShipDef[hullConfig.id].name,
+		shipname = ShipDef[hullConfig.id].modelName,
```

Comment:
It might be better to rename the variable to indicate to other developers that this is no longer the actual shipname, but the modelname which must then be used as a key to the translation.

Alternatively, why not perform the translation here which means you don't have to call the translation function everywhere else in this module?